### ISSUE TITLE
[Feature-Idea] Advertisement collecting

### ISSUE THREAD

**Issue by FaBjE** [sensboston/BLEConsole#33]
_Note: This is a new feature idea, I don't know if I will find the time to implement this, anyone is welcome to do so_

**Rationale**
More and more (low-power) devices publish their information in a advertisement message instead or in addition to the GATT.
Connecting to such a (battery powered) device will get you the info but will drain the battery of the device.

**Description**
It would be a good feature to be able to receive/collect these advertisement messages and provide them as output.

**Implementation**
.NET has the [BluetoothLEAdvertisementWatcher](https://learn.microsoft.com/en-us/uwp/api/windows.devices.bluetooth.advertisement.bluetoothleadvertisementwatcher) class for this purpose.
Basically creating start trigger and displaying the output from the receive event would implement this feature.

**Comment by sensboston** [comment]
Sound good, agree

**Comment by sensboston** [comment]
BTW, there is another BLE watcher available, called **BluetoothLEAdvertisementWatcher**
I already tested, it works stable enough.
